writeengine/wrapper/writeengine.cpp

```diff
@@ -6173,42 +6160,38 @@ void WriteEngineWrapper::RemoveTxnFromDictMap(const TxnID txnid)
     }
 }
 
+int WriteEngineWrapper::markTxnExtentsAsInvalid(const TxnID txnid, bool erase)
+{
+    int rc = 0;
+    std::tr1::unordered_map<TxnID, SP_TxnLBIDRec_t>::iterator mapIter = m_txnLBIDMap.find(txnid);
+    if (mapIter != m_txnLBIDMap.end())
+    {
+        SP_TxnLBIDRec_t spTxnLBIDRec = (*mapIter).second;
+        if (spTxnLBIDRec->m_LBIDs.size())
```

Comment:
`if (!spTxnLBIDRec->m_LBIDs.empty())` is probably a better check here.

Reply:
Done that.